Source/Core/Core/HW/DSPHLE/UCodes/AX.cpp

```diff
@@ -119,13 +119,6 @@ void AXUCode::HandleCommandList()
 
   u32 pb_addr = 0;
 
-#if 0
-	INFO_LOG_FMT(DSPHLE, "Command list:");
-	for (u32 i = 0; m_cmdlist[i] != CMD_END; ++i)
-		INFO_LOG_FMT(DSPHLE, "{:04x}", m_cmdlist[i]);
-	INFO_LOG_FMT(DSPHLE, "-------------");
-#endif
```

Comment:
This one _used to_ have a purpose: log the AX command list in case someone needed it; but only when compiled in. No strong feelings if we want to remove it though, it's easy enough to rewrite from scratch if necessary 👍